website/content/pro/copilot/index.md

```diff
@@ -0,0 +1,139 @@
+answer the user's query.
+
+It's important to understand which data was used by OpenBB Copilot to formulate
+an answer. As a result, OpenBB Copilot will cite which data source it utilized
+in order to answer the user's query.
+
+<ReactPlayer width="50%" height="100%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/2156f1cb-f49e-4481-8131-41fcd6b91672' />
+
+Since OpenBB Copilot can access data from any widget in the active dashboard, we
+encourage users to experiment with adding different kinds of widgets and
+experimenting with various queries. For example, OpenBB Copilot is particularly
+effective at summarizing earnings call transcripts from the "Earnings
+Transcripts" widget.
+
+## Querying specific widgets only
+
+<ReactPlayer width="70%" height="100%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/1335e310-cd65-4917-bc34-1de8b3e5f7fc' />
+
+
+Sometimes you may wish to focus your analysis and utilize OpenBB Copilot to analyze only
+ a specific subset of widgets. For example, you may want to use OpenBB Copilot
+ is assist you in a deep analysis of an earnings transcript in the "Earnings
+ Transcript" widget, without retrieving data from the rest of the dashboard.
+ 
+ To achieve this, you can chat with specifically selected widgets by clicking on
+ the "Add widgets as context" button on each widget you wish to use while
+ querying OpenBB Copilot. Selecting a widget in this manner will make that
+ widget's data available to OpenBB Copilot, while excluding other widgets that
+ have not been selected. You can then use OpenBB Copilot as normal, and the
+ unselected widgets in the rest of the dashboard will be ignored by OpenBB
+ Copilot.
+
+## Querying your own data
+
+<ReactPlayer width="70%" height="100%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/905eb674-5619-4797-8adf-9cf13a846792' />
+
+OpenBB Copilot can also answer queries using files that you provide. Currently
+TXT, PDF, CSV and XLSX files are supported. Files can be added to the Copilot by
+dragging and dropping them on the OpenBB Copilot chatbox, or by clicking the
+paper clip icon.
+
+Once your files have been uploaded, OpenBB Copilot will use the data in the
+uploaded files, if necessary, to answer your queries. If OpenBB Copilot uses the data
+contained in your files to answer a query, it will cite which files it used (and
+in the case of PDFs, the specific page).
+
+### A note on filenames
+OpenBB Copilot makes use of filenames to assess whether a file is
+relevant to the user's query. As a result, it is highly recommended that you use
+filenames that are descriptive of the data that they contain. For example, given
+a PDF file containing a technical report from TSLA released in 2024, a good
+filename would be `tsla_technical_report_2024.pdf`.
```

Comment:
not a complaint or blocking, but we could put this in one of those fun info boxes